app/views/pages/contact_us.html.erb

```diff
@@ -0,0 +1,85 @@
+<% if current_user.present? %>
+  <div class="govuk-!-padding-bottom-4">
+    <%= link_to 'Home', prison_dashboard_index_path(@user.active_case_load_id) %>
+  </div>
+<% end %>
+
+<% if notice.present? %>
+  <p class="notification govuk-!-margin-bottom-9" job_type="group" aria-labelledby="notice-heading">
+    <%= notice %>
+  </p>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-5">Help</h1>
+    <h2 class="govuk-heading-m">Getting set up</h2>
+    <p class="govuk-body">If you are new to this service, or have staff members you need to add, see the step by step
+      <%= link_to 'guide to getting set up', help_path %>
+    </p>
+  </div>
+</div>
+
+<div class="govuk-grid-row dashboard-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if @contact.errors.count > 0 %>
+      <%= render :partial => "/shared/validation_errors", :locals => { :errors => @contact.errors } %>
+    <% end %>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-form-group ">
+      <h2 class="govuk-heading-m">Contact us</h2>
+      <p class="govuk-body">Complete this form for technical help</p>
+
+      <%= form_tag(help_path, method: :post, id: "help_form") do %>
```

Comment:
path is wrong, help_path isn't the right place.

Reply:
I believe this has been addressed in recent commit